components/Chips/src/MDCChipView.h

```diff
@@ -29,6 +29,12 @@
 
  Chips contain an optional leading image, a title label and an optional trailing accessory view.
  They can also contain a leading image that appears only when the chip is selected.
+
+ Chips currently support two layout styles: default and centered. In the default mode, the image and
```

Comment:
Nit: I think it's better to reference the `contentHorizontalAlignment` property directly here.  Using Fill and Center instead of Default and Centered would also help clarify.

Reply:
Done.